src/app/core/admin/admin-entity-details/admin-entity-panel-component/admin-entity-panel-component.component.html

```diff
@@ -1,7 +1,35 @@
 <div class="padding-regular">
-  <p>[ {{ config.component }} ]</p>
+  @if (entityConstructor?.ENTITY_TYPE) {
+    <mat-form-field appearance="fill" class="margin-bottom-regular">
+      <mat-label i18n>Entity Type</mat-label>
+      <app-entity-type-select
+        [(ngModel)]="selectedEntityType"
+        (ngModelChange)="onEntityTypeChange($event)"
+        [multi]="false"
+        [showInternalTypes]="true"
```

Comment:
I think I would only show the "not internal" entity types to users. What have your thoughts been on including internal ones?